evap/static/scss/components/_notebook.scss

```diff
@@ -43,42 +43,43 @@
         flex-grow: 1;
     }
 
-    @media screen and (min-aspect-ratio: $notebook-break) {
-        .notebook-container {
-            width: 25vw;
-            padding-left: $spacer;
-            padding-bottom: $spacer;
-            padding-top: $spacer;
-        }
+    display: flex;
+    justify-content: center;
+    .notebook-container {
+        width: 80vw; //approximation to be similar to evap-Content width
```

Comment:
Can we use the exact same value as we use with evap-content? I feel like there should be a way to factor that out into a variable, again, and reuse it here.